djangobench/benchmarks/query_values_10000/benchmark.py

```diff
@@ -0,0 +1,19 @@
+from djangobench.utils import run_benchmark
+
+
+def setup():
+    global Book
+    from query_values_10000.models import Book
+    Book.objects.bulk_create((
+        Book(title='title')
+        for x in range(10000)
+    ))
+
+def benchmark():
+    global Book
+    for x in Book.objects.values('title'): pass
```

Comment:
`list(Book.ob...)` is the most standard way of no-op iterating a QS in django tests I think

Reply:
I did it this way to exclude list copying from benchmark, but I think `bool(qs)` would be even better.

Reply:
it's not deep-copying though, `for x in` is exercising the garbage collection pretty much the same amount